Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -120,6 +124,7 @@ import BraintreeDataCollector
         completion: @escaping (BTPayPalAccountNonce?, Error?) -> Void
     ) {
         tokenize(request: request, completion: completion)
+        isVaultRequest = false
```

Comment:
Should we default `isVaultRequest` to `false` so we don't have to set it to false in all non-vaulting cases?

Reply:
Maybe I don't understand the PayPal flow well enough. Could there be a situation where a user checks out with Vault but doesn't complete transaction and checks out with 1 time checkout in the same transaction?

Reply:
🤔 That would certainly be an edge case, but probably a good call to handle it incase it does happen!